src/web-errors.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpStatus } from "@fwl/web";
+import {
+  ApplicationError,
+  WebError,
+  WebErrorDetails,
+} from "@fwl/web/dist/errors";
+
+const ERRORS_DATA = {
+  BAD_REQUEST: {
+    code: "unexpected_error",
+    httpStatus: HttpStatus.BAD_REQUEST,
+    message: "Request malformed",
+  },
+  UNEXPECTED_ERROR: {
+    code: "unexpected_error",
```

Comment:
Should the two errors have the same code ? I understood the code property as being unique for each error 🤔

Reply:
My bad 🤦.